DeviceAdministration/Infrastructure/BusinessLogic/DeviceLogicND.cs

```diff
@@ -1156,14 +1156,6 @@ private static IEnumerable<DevicePropertyMetadata> GetDevicePropertyConfiguratio
                 PropertyType = PropertyType.Status
             };
 
-            yield return new DevicePropertyMetadata()
-            {
-                IsDisplayedForRegisteredDevices = false,
-                IsDisplayedForUnregisteredDevices = false,
-                IsEditable = false,
-                Name = "HostName"
```

Comment:
why was this even there ? can you check if this field is used anywhere else in the solution ?